app/models/message.rb

```diff
@@ -0,0 +1,3 @@
+class Message < ActiveRecord::Base
+  belongs_to :recipient
```

Comment:
According to the messages entry in the wiki I thought that channel would be a string identifying what channel(s) the message was sent on. I see now that there's a model for channels, I apologize for my oversight, I'll correct this.